common/libs/wallet/base.js

```diff
@@ -1,11 +1,15 @@
 // @flow
 
 export default class BaseWallet {
-  getAddress(): string {
+  getAddress(): Promise<any> {
```

Comment:
If it's going to get implemented as a promise elsewhere, this should probably return a promise that rejects, rather than throwing.